src/main/java/de/dennisguse/opentracks/io/file/ErrorListDialog.java

```diff
@@ -0,0 +1,35 @@
+package de.dennisguse.opentracks.io.file;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.os.Bundle;
+
+import androidx.fragment.app.DialogFragment;
+
+import java.util.List;
+
+import de.dennisguse.opentracks.R;
+
+public class ErrorListDialog extends DialogFragment {
+
+    public static final String TAG = ErrorListDialog.class.getSimpleName();
+
+    private List<String> trackErrors;
+    private String title;
+
+    public ErrorListDialog(String title, List<String> trackErrors) {
```

Comment:
is it sufficient to pass parameters via the constructor?
If the activity will be restored all fragments should be created from a bundle.
Like this: 
```

    public static void showDialog(FragmentManager fragmentManager, Uri documentFileUri, boolean isDirectory) {
        Bundle bundle = new Bundle();
        bundle.putParcelable(EXTRA_DATA_URI_KEY, documentFileUri);
        bundle.putBoolean(EXTRA_IS_TREE_KEY, isDirectory);

        ImportProgressDialogFragment dialogFragment = new ImportProgressDialogFragment();
        dialogFragment.setArguments(bundle);
        dialogFragment.setRetainInstance(true);
        dialogFragment.show(fragmentManager, IMPORT_DIALOG_TAG);
    }
```

Reply:
You're right. What a screw up! I didn't test it :see_no_evil: